modules/kubernetes/goldpinger/templates/values.yaml.tpl

```diff
@@ -0,0 +1,7 @@
+goldpinger:
+  port: 8080
+
+podAnnotations:
+  linkerd.io/inject: enabled
```

Comment:
make this optional and based on the linkerd_enabled var

Reply:
Done